src/task/src/pullRequests/pullRequest.ts

```diff
@@ -15,17 +16,20 @@ export default class PullRequest {
   private readonly _codeMetrics: CodeMetrics
   private readonly _logger: Logger
   private readonly _runnerInvoker: RunnerInvoker
+  private readonly _inputs: Inputs
 
   /**
    * Initializes a new instance of the `PullRequest` class.
    * @param codeMetrics The code metrics calculation logic.
    * @param logger The logger.
    * @param runnerInvoker The runner invoker logic.
+   * @param inputs The inputs passed to the task
```

Comment:
Parameters are ordered alphabetically for consistency. Please do the same for this new parameter. Applies to the class variables above as well.